### PR TITLE
RVS json logging for iet,gst,pebb and corrections

### PR DESCRIPTION
**Pull request by manoj-freyr**
Cleanup and remove redundant info from json logs generated.
Enable changes for pebb reporting

### CODE REVIEW

**Review by ashutom — COMMENTED** [review]
It smells that this is a good opportunity to apply "visitor pattern" here. Have you thought of it?

**Comment by jkottiku** [comment]
General comment - please check and correct the code indentation.

**Review by jkottiku — CHANGES_REQUESTED** [review]

**Review by jkottiku — APPROVED** [review]

File: pebb.so/include/action.h [hunk 1]
```diff
@@ -105,6 +105,12 @@ class pebb_action : public rvs::actionbase {
                       uint32_t Distance,
                       const std::vector<rvs::linkinfo_t>& arrLinkInfo,
                       bool bReverse);
+  void json_add_primary_fields();
```
[the remark this answers is not in the document]
Comment (manoj-freyr) [reply pebb.so/include/action.h:108]: this is under assumption that they can be changed independently when we make changes to each module reporting. as of now iet and gst looks similar but they can change independently and that is why it is kept with modules

**Comment by manoj-freyr** [comment]
> It smells that this is a good opportunity to apply "visitor pattern" here. Have you thought of it?

Wil take it up in next iteration. This was more of removing redundant items from reported json and adding changes to pebb

**Pull request merged**